server/tests/src/test/resources/proto/v1.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package evolution;
+
+enum e1 {
+  V0 = 0;
+  V1 = 1;
+  V2 = 2;
+
+  reserved 3, 4, 5;
+  reserved "V3", "V4";
+}
+
+message m1 {
+  string f1 = 1;
+  int32 f2 = 2;
+  bool f3 = 3;
+  map<string,string> f4 = 4;
+
+  reserved 5, 6, 7;
+  reserved "f5", "f6";
+}
```

Comment:
Is this file used by some test? (also the other in the same folder)

Reply:
I originally wanted to have these in the server testsuite, but it was easier to run/debug in the rest testsuite :). I'll remove them